expr.go

```diff
@@ -50,7 +51,9 @@ func fillScope(vars *stateful.Scope, referenceVariables []string, now time.Time,
 		}
 		if !isFieldExists && !isTagExists {
 			if !vars.Has(refVariableName) {
-				return fmt.Errorf("no field or tag exists for %s", refVariableName)
+				var missingValue interface{} = &ast.Missing{}
```

Comment:
Should the ast package export a VarMissing var so that we do not have to allocate a new one each time?

```
 vars.Set(refVariableName, ast.MissingValue)
```